src/renderers/shaders/UniformsLib.js

```diff
@@ -18,6 +18,7 @@ var UniformsLib = {
 		offsetRepeat: { value: new Vector4( 0, 0, 1, 1 ) },
 
 		specularMap: { value: null },
+		uSpecularStrength: { value : 1 },
```

Comment:
This is just an issue of a map, missing the complementary parameter. Happens to have the term "specular" in its name. I dont understand why lighting equations and physics are being discussed.